naturf/tests/test_nodes.py

```diff
@@ -105,6 +105,50 @@ class TestCase:
                 actual,
                 f"failed test {case.name} expected {expected}, actual {actual}"
             )
+    
+    def test_wall_angle_direction_length(self):
+    """Test that the function wall_angle_direction_length returns the correct angle, direction, and length."""
+    
+    polyext = [[0,1], [1,1], [1,0], [0,0], [0,1]]
```

Comment:
I'd give line 112 and 113 more descriptive names. guessing it's meant to mean `polygon_exterior` and `polygon_interior`?